tests/topotests/lib/common_config.py

```diff
@@ -3012,159 +3010,157 @@ def verify_rib(
                         st_rt = str(ipaddress.ip_network(frr_unicode(st_rt)))
 
                         _addr_type = validate_ip_address(st_rt)
-                        if _addr_type != addr_type:
-                            continue
-
-                        if st_rt in rib_routes_json:
-                            st_found = True
-                            found_routes.append(st_rt)
-
-                            if fib and next_hop:
-                                if type(next_hop) is not list:
-                                    next_hop = [next_hop]
+                        if _addr_type == addr_type:
```

Comment:
Because this change leads to a great deal of indentation change, it's very hard to see what else might have changed. If there was no other change, why not leave the 'continue' in place? If there was another change, please highlight that.